CHANGES.md

```diff
@@ -14,6 +14,11 @@ limits on path length.
 - Shorten the length of the name of the cache file to fix crashes on file systems that
   do not support long paths (#4176)
 
+### Output
```

Comment:
Oops I forgot to add a new Unreleased section yesterday. I'll do that now and then move this up into the right section.